src/javascripts/components/hangar/hangar.js

```diff
@@ -2,10 +2,78 @@ import planesData from '../../helpers/data/planesData';
 import utils from '../../helpers/utils';
 import planeComponent from '../plane/plane';
 
+const createPlane = (e) => {
+  e.preventDefault();
+  const newPlane = {
+    imgageUrl: $('#plane-image').val(),
```

Comment:
your images are not working because of this key = spelled image wrong.

should be imageUrl